lib/uffizzi/services/compose_file_service.rb

```diff
@@ -14,8 +14,44 @@ def parse(compose_content, compose_file_path)
       prepare_dependencies(env_files, config_files, compose_file_path)
     end
 
+    def process_env_variables(compose_file_data)
+      compose_file_data.gsub(/\$\{?([?:\-_A-Za-z0-9]+)\}?/) do |var|
+        var_content = var.match(/[?:\-_A-Za-z0-9]+/).to_s
```

Comment:
What does `content` mean? Is it variable name?

Reply:
`content` - is everything inside brackets
for example - content for `${PORT:-80}` is `PORT:-80`